.ci/pitest.sh

```diff
@@ -4,12 +4,22 @@ set -e
 
 ###############################
 function checkPitestReport() {
-  ignored=("$@")
+  local -n ignored=$1
   fail=0
   SEARCH_REGEXP="(span  class='survived'|class='uncovered'><pre>)"
   grep -irE "$SEARCH_REGEXP" target/pit-reports \
      | sed -E 's/.*\/([A-Za-z]+.java.html)/\1/' | LC_ALL=C sort > target/actual.txt
   printf "%s\n" "${ignored[@]}" | sed '/^$/d' | LC_ALL=C sort > target/ignored.txt
+  if [ -n "$2" ] ; then
+      local -n unstableMutations=$2
```

Comment:
I do not see usage of second parameter in call of pitest.sh

Reply:
Yes, I am adding it, was re-running CI to get the latest result.